apps/web/lib/features/team/user-team-card/task-skeleton.tsx

```diff
@@ -43,29 +43,23 @@ export function InviteUserTeamSkeleton() {
 export function UserTeamCardHeader() {
 	const t = useTranslations();
 	return (
-		<div className="hidden sm:flex row font-normal justify-between pb-5 pt-8 hidde dark:text-[#7B8089]">
-			{/* <li className="pr-[50px]">{t('common.STATUS')}</li> */}
-			<div className="2xl:w-[20.625rem] text-center">{t('common.NAME')}</div>
-			<div className="w-1"></div>
-			<div className="2xl:w-80 3xl:w-[32rem] w-1/5 text-center">{t('common.TASK')}</div>
-			<div className="w-1"></div>
-			<Tooltip label={t('task.taskTableHead.TOTAL_WORKED_TODAY_HEADER_TOOLTIP')}>
-				<div className="2xl:w-48 3xl:w-[12rem] w-1/5 lg:px-4 px-2 flex flex-col justify-center text-center">
-					{t('task.taskTableHead.TASK_WORK.TITLE')}
-					<br />
-					{t('common.TASK')}
-				</div>
-			</Tooltip>
-			<div className="w-1"></div>
-			<div className=" text-center w-1/5 lg:px-3 2xl:w-52 3xl:w-64">{t('common.ESTIMATE')}</div>
-			<div className="w-1"></div>
-			<Tooltip label={t('task.taskTableHead.WORKED_ON_TASK_HEADER_TOOLTIP')}>
-				<div className="w-1/5 lg:px-3 2xl:w-52 3xl:w-64 text-center">
-					{t('task.taskTableHead.TOTAL_WORK.TITLE')}
-					<br />
```

Comment:
@Anishali2 how you replaced that <br>?

Reply:
Sir I cannot remove something in this page I just created the responsive ness from scratch and put the under code in there because it is more easy , fast , and relaible 
also understanding every single class and change will take a lot of time.